Guzzle/RequestNewAccessToken.php

```diff
@@ -17,6 +17,14 @@ public function __construct(ClientInterface $httpClient)
     }
 
 
+    /**
+     * @param string $clientId
+     * @param string $clientSecret
+     * @param string $redirectUri
+     * @param string $refreshToken
+     *
+     * @return array
```

Comment:
Would be better to return some object like `return new AccessToken($accessToken, $refreshToken)` returning an array should be avoided